server/settings.py

```diff
@@ -95,6 +96,13 @@
     },
 ]
 
+WIRE_AGGS = {
+    "genre": {"terms": {"field": "genre.name", "size": 50}},
+    "_subject": {
```

Comment:
You can just use `subject` as it is in the default config, as the system will automatically rename it when nested subject groups are loaded.

Reply:
yep I was playing with that, I was trying to actually get rid of it because we don't need it, but seems like it's required